modules/service/src/test/scala/lucuma/odb/graphql/query/executionFailures.scala

```diff
@@ -104,21 +107,60 @@ class executionFailures extends ExecutionTestSupport {
                }
              }
            """,
-        expected = Ior.both(
-          List(s"Could not generate a sequence from the observation $oid: observing mode"),
-          json"""
-            {
-              "observation": {
-                "execution": {
-                  "config": null
-                }
-              }
-            }
-          """
+        expected = {
+          case OdbError.SequenceUnavailable(Some("Could not generate a sequence from the observation o-101: observing mode")) => //ok
+        }
+      )
+    }
+  }
+
+/*
+  In this test case, the query requests both the observation `execution` field
+  and the `itc` field.  Neither query should be successful because there is no
+  observing mode.  We expect two errors, one for `execution` and one for `itc`
+  and often actually get them.  Sometimes though only the `itc` error is
+  returned.
```

Comment:
This would seem to be an issue with Grackle?

Reply:
This is weird because Grackle only demands `MonadThrow` for its effect type, which means it shouldn't be able to do anything concurrently and results should be deterministic. Could this test be interacting nondeterministically with others in the suite? Maybe move it to its own suite so it gets its own database?

Reply:
Thanks for the suggestion.  I isolated the test case into `lucuma.odb.graphql.issue.shortcut.3660`.  Upon attempting it I got 3 consecutive failures, followed by 5 success, followed by 1 failure.  All the failures mss the same error:

```
values are not the same
=> Obtained
Set(
  InvalidObservation(
    observationId = Id(
      value = 256
    ),
    detail = Some(
      value = "ITC cannot be queried until the following parameters are defined: observing mode."
    )
  )
)
=> Diff (- obtained, + expected)
 Set(
+  SequenceUnavailable(
+    detail = Some(
+      value = "Could not generate a sequence from the observation o-100: observing mode"
+    )
+  ),
```